### PR TITLE
Enable more printer tests on unix and improve testing

### PR DESCRIPTION
**Pull request by hughbe**

### CODE REVIEW

File: src/System.Drawing.Common/src/System/Drawing/Printing/Margins.cs [hunk 1]
```diff
@@ -91,10 +94,10 @@ public int Right
         /// </summary>
         public int Top
         {
-            get { return _top; }
+            get => _top;
             set
             {
-                CheckMargin(value, "Top");
+                CheckMargin(value, nameof(value));
```
Comment (danmosemsft) [inline src/System.Drawing.Common/src/System/Drawing/Printing/Margins.cs]: Are we consistent about what we use as parameter name in argument exceptions from properties? I've seen `value` often used like here, but in my mind the name of the property is more useful. `value` is merely the C# keyword.

**Comment by danmosemsft** [comment]
thanks!

File: src/System.Drawing.Common/src/System/Drawing/Printing/Margins.cs (hunk 1 above)
Comment (danmosemsft) [reply src/System.Drawing.Common/src/System/Drawing/Printing/Margins.cs]: @stephentoub @JeremyKuhne any preference?

Comment (stephentoub) [reply src/System.Drawing.Common/src/System/Drawing/Printing/Margins.cs]: We've generally decided on using "value", which is not only the C# keyword but also the name of the set_* argument in the IL.  I don't have a strong preference, other than to not churn a lot of code to change it.

**Pull request merged**